Samples/Simple/MediaSample.Droid/MainActivity.cs

```diff
@@ -56,7 +56,7 @@ protected override void OnCreate(Bundle savedInstanceState)
 
             if (ShouldUseExoPlayer)
             {
-                ((MediaManagerImplementation)CrossMediaManager.Current).MediaSessionManager = new MediaSessionManager(Application.Context, typeof(ExoPlayerAudioService), MediaManager);
+                ((MediaManagerImplementation)CrossMediaManager.Current).MediaSessionManager = new MediaSessionManager(Application.Context, MediaManager);
```

Comment:
@nickrandolph otherwise the project doesn't build. The constructor of MediaSessionManager has been changed by someone (in previous commits), to only take 2 parameters. I simply removed this parameter to make the project build